File: Programming-languages-and-laboratory/Java/Exercises/38_compito_java_2020_02_04/Cliente.java
import java.util.*;

public class Cliente {

	private String nome, cognome, indirizzo;
	private int codice;
	protected List<Prenotazione> prenotazioni;

	public Cliente(int c, String n, String cog, String i) {
		codice = c;
		nome = n;
		cognome = cog;
		indirizzo = i;
		prenotazioni = new LinkedList<Prenotazione>();
	}

	public void addPrenotazione(Prenotazione p) {
		prenotazioni.add(p);
	}

	public String toString() {
		return codice + "\t" + cognome + "\t" + nome + "\t" + indirizzo + "\t" + prenotazioni;
	}
}

File: II - Linguaggi di programmazione e laboratorio/Python/Exercises/38_compito_python_20200204/Cliente.py
class Cliente:
    def __init__(self, codice, nome, cognome, indirizzo):
        self._codice = codice
        self._nome = nome
        self._cognome = cognome
        self._indirizzo = indirizzo
        self._noleggi = []

    def add_noleggio(self, nol):
        self._noleggi.append(nol)

    def __str__(self):
        return str(self._codice) + "\t" + self._cognome + "\t" + self._nome + "\t" + self._indirizzo + "\t" + \
               str([str(nol) for nol in self._noleggi])
